Three_Sum_Test.java

```diff
@@ -0,0 +1,39 @@
+package Week_2;
+import java.util.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+class Three_Sum_Test {
+
+	@Test
+	void test1() {

```

Comment:
What are these test cases testing? It can be helpful and easier to read if the test method names indicate what they are testing for instead of just test1, test2, test3. For example, this one could be testEmptyArray